lib/tasks/db.rake

```diff
@@ -59,5 +59,17 @@ namespace :db do
     end
   end
 
+  namespace :index do
+    desc "Creates indexes for mango querys located in db/indexes/*.json"
+    task create: :environment do
+      indexes_dir = Rails.root.join('db', 'indexes')
+      files = Dir.glob File.join(indexes_dir, '**', '*.json')
+
+      files.each do |file|
+        index_data = JSON.parse(File.read(file))
+        Dolly::MangoIndex.create(index_data['name'], index_data['fields'])
```

Comment:
I wonder if the freeze string literals direction works with rake files 🤔

Reply:
it does work i think, i recall rubocop throws a warning in rake files to add it